core/scripts/Voting.js

```diff
@@ -144,6 +224,78 @@ async function fetchIntrinioEquitiesPrice(request, config) {
   return web3.utils.toWei(price.toString());
 }
 
+async function fetchIntrinioForexPrice(request, config) {
+  const url = [
+    "https://api-v2.intrinio.com/forex/prices/",
+    config.symbol,
+    "/m1?",
+    "api_key=" + process.env.INTRINIO_API_KEY
+  ]
+    .concat(getIntrinioTimeArguments(request.time))
+    .join("");
+  console.log(`\n    ***** \n Querying with [${stripApiKey(url, process.env.INTRINIO_API_KEY)}]\n    ****** \n`);
+  const jsonOutput = await getJson(url);
+
+  if (!jsonOutput.prices || jsonOutput.prices.length === 0) {
+    // The JSON output can be large when it succeeds, so we only print it in cases of failure.
+    console.log("Intrinio response:", jsonOutput);
+    throw "Failed to get data from Intrinio";
+  }
+
+  // TODO(ptare): Forex quotes don't appear to have trade prices!?
+  const price = jsonOutput.prices[0].open_bid;
+  const time = jsonOutput.prices[0].occurred_at;
+  console.log(`Retrieved quote [${price}] at [${time}] for asset [${web3.utils.hexToUtf8(request.identifier)}]`);
+  return web3.utils.toWei(price.toString());
+}
+
+async function fetchIntrinioCryptoPrice(request, config) {
+  const url = [
+    "https://api-v2.intrinio.com/crypto/prices?",
+    "api_key=" + process.env.INTRINIO_API_KEY,
+    "&currency=" + config.symbol,
+    "&timeframe=m1"
+  ]
+    .concat(getIntrinioTimeArguments(request.time))
+    .join("");
+  console.log(`\n    ***** \n Querying with [${stripApiKey(url, process.env.INTRINIO_API_KEY)}]\n    ****** \n`);
+  const jsonOutput = await getJson(url);
+
+  if (!jsonOutput.prices || jsonOutput.prices.length === 0) {
+    // The JSON output can be large when it succeeds, so we only print it in cases of failure.
```

Comment:
nit: You can use the parameter `page_size=1` so the response only returns a single element rather than the default of 100.